docs/adr/2025-12-23_003-sqlite.md

```diff
@@ -0,0 +1,37 @@
+---
+slug: 3
+title: | 
+  3. SQLite
+authors: [tweag-ev-ak]
+tags: [Proposed]
+---
+
+## Status
+
+Proposed
+
+## Context
+
+- We want to store the results of scripts execution for future usage.
+
+## Decision
+
+- We decided to use SQLite because:
+
+    1. The whole database is just a file, no need to add one more running service.
+
+    2. Familiar query language
+
+- We decided to use `sqlite-simple` Haskell library:
+
+    1. To keep the number of dependencies small.
+
+    2. It's easy to easy and allows to write SQL queries manually. 
```

Comment:
Just because it sounds like we're creating queries by concatenating strings (we're totally not doing that...)
```suggestion
    2. It's easy to easy and allows to write SQL queries manually. 
    
    3. It supports parameterised queries, avoiding SQLi issues.
 ```